portfolio/src/main/webapp/script.js

```diff
@@ -29,10 +29,23 @@ function addRandomFacts() {
 
 /** Request content from server and add it to page */
 function getServerMessage() {
-    fetch('/data')
-    .then(response => response.text())
-    .then((fruits) => document.getElementById('server-message-container').innerHTML = fruits)
+    fetch('/data').then(response => response.json()).then((comments) => {
+        const history = document.getElementById('server-message-container');
```

Comment:
You're adding <li> tags, so this needs to be commentsList not server-message-container.  You want to end up with
`<ul>`
`  <li>...</li>`
`  <li>...</li>`
`  ...`
`</ul>`

The current code would append to the <div> so you get:
`<div>`
`  <ul></ul>`
`  <li>...</li>`
`  <li>...</li>`
`  ...`
`</div>`

Reply:
That makes sense, thank you for the feedback!